social-network/src/articles-missed.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+

```

Comment:
I could not find any real differences between this component and articles-yours.js 
When components share this much similarities you might want to consider combining them. If the changes are minimal such as an extra HTML tag etc you could always pass some sort of flag as a prop and conditionally render the extra code.